libkbfs/tlf_journal.go

```diff
@@ -460,6 +479,10 @@ func (j *tlfJournal) flushOneBlockOp(ctx context.Context) (bool, error) {
 		return false, err
 	}
 
+	// TODO: There is a race condition if we shut down after we've
+	// flushed an op but before we remove it. Make sure we handle
+	// re-flushed ops idempotently.
```

Comment:
This TODO is basically just to verify that BServer ops are all idempotent. We talked about this, and they should be, right?